expo/tests/scheduler/test_delete_schedule.py

```diff
@@ -0,0 +1,76 @@
+from django.core.urlresolvers import reverse
+from django.test import (
+    Client,
+    TestCase,
+)
+from tests.factories.gbe_factories import (
+    ProfileFactory,
+)
+from scheduler.models import EventItem
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    is_login_page,
+    login_as,
+)
+from tests.contexts import ClassContext
+from gbe.models import Conference
+
+
+class TestDeleteSchedule(TestCase):
+    view_name = 'delete_schedule'
+
+    def setUp(self):
+        self.client = Client()
+        self.user = ProfileFactory.create().user_object
+        self.privileged_profile = ProfileFactory()
+        self.privileged_user = self.privileged_profile.user_object
+        grant_privilege(self.privileged_user, 'Scheduling Mavens')
+        Conference.objects.all().delete()
+        self.context = ClassContext()
```

Comment:
This worries me a little, since now the context is shared among all of the tests in this TestCase. This means that changes to it are shared, which could lead to test dependencies. For example, if someone in the future makes a change to self.context.performer, that would affect any existing tests that happened to run after that one